lib/bookshelf.js

```diff
@@ -30,149 +30,162 @@ function Bookshelf(knex) {
     throw new Error('Invalid knex instance');
   }
   const bookshelf = {
-    VERSION: require('../package.json').version
+    VERSION: require('../package.json').version,
   };
 
-  const Model = bookshelf.Model = BookshelfModel.extend({
-    _builder: builderFn,
+  const Model = (bookshelf.Model = BookshelfModel.extend(
+    {
```

Comment:
I'm guessing we can't keep this `{` in the previous line, right?

Reply:
Your guess it right

Reply:
Ok, part of the compromise part I guess. Not a big deal. Thanks for your comments.